cmd/kgo-verifier/main.go

```diff
@@ -266,7 +266,7 @@ func main() {
 		}
 
 		firstPass := true
-		for firstPass || (len(lastPassChan) == 0 && *loop) {
+		for firstPass || (len(lastPassChan) != 0 && *loop) {
```

Comment:
Wouldn't this just loop forever after last_pass is called?

Reply:
Copy & paste what I did for other cases https://github.com/redpanda-data/kgo-verifier/pull/48/files?

Reply:
will do it